docs/topics/development.rst

```diff
@@ -0,0 +1,59 @@
+Development
+===========
+
+Here we describe the development process overview. It's in F.A.Q. format to
+make it simple.
+
+
+How to file a ticket?
+---------------------
+
+Just go to https://github.com/coddingtonbear/django-mailbox and create new
+one.
+
+
+How do I get involved?
+----------------------
+
+It's simple! If you want to fix a bug, extend documentation or whatever you
+think is appropriate for the project and involves changes, just fork the
+project at github (https://github.com/coddingtonbear/django-mailbox), create a
```

Comment:
Probably would use 'on' rather than 'at'.  I'm honestly not sure what the fundamental rule is, but I think we generally do things "on" a website, but "at" a URL.  Thinking this through, it reminds me of the giant chart I once had to memorize for when to use в vs на when speaking Russian.